### PR TITLE
Removes legacy TravisCI config files and updates relevant docs

### PR DESCRIPTION
**Pull request by dvail**
## Description

With the move to containerized frontend builds handled by RH Jenkins, Travis CI is no longer needed. This changes removes the stale config files.

## Test manual

**TODO:** Add manual testing efforts

```
# To run tests locally run:
npm run test
```

### CODE REVIEW

**Comment by dvail** [comment]
@alwayshooin  @bradr5 Do you know if there is anything external we need to do to deprecate Travis as well? Or is removing these files sufficient?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,51 +10,20 @@
...
+- staging (preview) -> https://console.dev.redhat.com/preview/openshift/acs
+- staging -> https://console.dev.redhat.com/openshift/acs
+- prod (preview) -> https://console.redhat.com/preview/openshift/acs
+- prod -> https://console.redhat.com/openshift/acs
 
 ## Release Cycle
 
-@TODO: Answer the following questions: When should they merge code into the main, stable, prod-beta, and prod-stable branches? Do all of those branches require PRs with approval to make changes?
\ No newline at end of file
+@TODO: Answer the following questions: When should they merge code into the main, stable, prod-beta, and prod-stable branches? Do all of those branches require PRs with approval to make changes?
```
Comment (alwayshooin) [inline CONTRIBUTING.md:29]: i think you can remove this line since we won't have to independently maintain non `main` branches anymore?

**Review by alwayshooin — APPROVED** [review]
from what i can tell, this should be sufficient on removing travis CI related items in the ACSCS repo. @bradr5 or @sachaudh is probably good to double check since they set up the flow initially

File: CONTRIBUTING.md (hunk 1 above)
Comment (dvail) [reply CONTRIBUTING.md]: Good catch. I thought I read that we might be moving to an actual release process for acs-ui in the future but we can remove this entire section until that becomes standardized.

**Pull request merged**